### DISCUSSION TITLE
Welcome to commanddotnet Discussions!

### DISCUSSION THREAD

**Discussion by bilal-fazlani** [bilal-fazlani/commanddotnet#325] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drewburlingame** [comment]
Hi all, I'm one of the contributors to this repo.  I'm happy to help if you have questions or would like advice or a review of how you're using CommandDotNet in your project.

**Comment by Nutr1k** [comment]
How to use AppSettings.Help.TextStyle = HelpTextStyle.Basic?
Can you give me an example?

**Reply by drewburlingame** [reply]
Hi @Code-Nostra, I'm not sure what you're asking for since that's how you'd use it.  https://commanddotnet.bilal-fazlani.com/help/help/#textstyle explains the difference, which is minor.  The code for it is https://github.com/bilal-fazlani/commanddotnet/blob/master/CommandDotNet/Help/BasicHelpTextProvider.cs which show's the overridden methods.

If you have a more specific question or there's some behavior you're trying to achieve, please create a new discussion.

Cheers,
Drew
